### PR TITLE
88 update documentation

### PR DESCRIPTION
**Pull request by maho3**
Updates all the documentation.

Also adds code coverage flags to the README.

### CODE REVIEW

**Comment by maho3** [comment]
A lot of files have changed, but the biggest changes are:

- Major updates to the main README.md
- All instances of 'summaries' when referring to model inputs have now been renamed 'data' (to be consistent with paper)

**Comment by maho3** [comment]
@all-contributors please add @CompiledAtBirth for code, design, and reviewing pull requests

**Comment by maho3** [comment]
@all-contributors
please add @CompiledAtBirth for code, design, review, content
please add @maho3 for infra, content, research
please add @DeaglanBartlett for infra, content, research
please add @florpi for research
please add @axellapel for code, research, examples
please add @shivampcosmo for research, examples
please add @christopherlovell for research, examples, data, content
please add @laperezNYC for research, content
please add @tlmakinen for code, research

**Comment by maho3** [comment]
@all-contributors please add @DeaglanBartlett for infra, content, research

**Comment by maho3** [comment]
@all-contributors please add @florpi for research

**Comment by maho3** [comment]
@all-contributors please add @axellapel for code, research, examples

**Comment by maho3** [comment]
@all-contributors please add @christopherlovell for research, examples, data, content

**Comment by maho3** [comment]
@all-contributors please add @shivampcosmo for research, examples

**Comment by maho3** [comment]
@all-contributors please add @laperezNYC for research, content

**Comment by maho3** [comment]
@all-contributors please add @tlmakinen for code, research

File: README.md [hunk 1]
```diff
@@ -1,27 +1,63 @@
...
+  * Jupyter and command line interfaces
+  * A parallelizable configuration framework for efficient hyperparameter tuning and production runs
+
+
+For more details on the motivation, design, and theoretical background of this project, see the software release paper ([arxiv:XXXXX](.)).
+
+
+
+## Getting Started 
+To install LtU-ILI, follow instructions in [INSTALL.md](INSTALL.md).
```
Comment (DeaglanBartlett) [inline README.md:46]: follow *the* instructions

File: ili/dataloaders/loaders.py [hunk 2]
```diff
@@ -80,11 +85,11 @@ def get_all_parameters(self):
 
 
 class StaticNumpyLoader(NumpyLoader):
-    """Class to load single numpy files of summaries and parameters
+    """Loads single numpy files of data and parameters from file
```
Comment (DeaglanBartlett) [inline ili/dataloaders/loaders.py:88]: Repeated "files" in sentence - could be worded better

**Review by DeaglanBartlett — COMMENTED** [review]
Very nice work! I have left a couple of comments where there may be typos

File: README.md (hunk 1 above)
Comment (maho3) [reply README.md:46]: Fixed.

File: ili/dataloaders/loaders.py (hunk 2 above)
Comment (maho3) [reply ili/dataloaders/loaders.py:88]: Fixed.

**Review by DeaglanBartlett — APPROVED** [review]

**Pull request merged**